DqtScripts/DqtScripts/Program.cs

```diff
@@ -96,10 +97,232 @@ static RootCommand CreateRootCommand()
     RevertAllowPiiUpdates(rootCommand);
     CorrectInductionStartDates(rootCommand);
     BackfillTrnRequestEmails(rootCommand);
+    ExportInductionPeriods(rootCommand);
 
     return rootCommand;
 }
 
+static void ExportInductionPeriods(RootCommand rootCommand)
+{
+    var command = new Command("export-induction-periods", description: "Exports induction periods, person & Appropriate bodies.")
+    {
+        Handler = CommandHandler.Create<IHost>(async (host) =>
+        {
+            var serviceClient = host.Services.GetRequiredService<ServiceClient>();
+            var blobContainerClient = host.Services.GetRequiredService<BlobContainerClient>();
+
+#if DEBUG
+            await blobContainerClient.CreateIfNotExistsAsync();
+#endif
+
+            var now = $"{DateTime.Now:yyyyMMddHHmmss}";
+            var inductionPeriodsFileName = $"induction-periods/inductionperiods_{now}.csv";
```

Comment:
It would probably be easier to handle these files if we put all three in a folder of their own (with `now` as the folder name)